src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -8135,12 +8135,11 @@ public static String strip(String str, final String stripChars) {
      */
     // See also Lucene's ASCIIFoldingFilter (Lucene 2.9) that replaces accented characters by their unaccented equivalent (and uncommitted bug fix: https://issues.apache.org/jira/browse/LUCENE-1343?focusedCommentId=12858907&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#action_12858907).
     public static String stripAccents(final String input) {
-        if (input == null) {
-            return null;
+        if (StringUtils.isEmpty(input)) {
```

Comment:
You don't need a FQCN here.

Reply:
You are correct, I pasted from text.CaseUtils so an empty string wouldn't have to be processed, and didn't think about the fact I was pasting it _into_ lang.StringUtils. I pushed the fix.